test/encodeQueryString.test.ts

```diff
@@ -0,0 +1,30 @@
+import {expect, test} from 'vitest'
+
+import encode from '../src/data/encodeQueryString'
+
+test('can encode basic query without parameters', () => {
+  const query = 'gamedb.game[maxPlayers == 64]'
```

Comment:
Woof, these are some ancient queries 😅 
Might send a PR to replace these with some moe modern GROQ at some point ;)